webapp/app/partials/perf/perfctrl.html

```diff
@@ -37,6 +37,10 @@
     </select>
     <hr/>
     <div id="graph"></div>
+    <div id="graph-bottom" ng-show="showGraphBottom">
+        Highlight revision: <input type="text" ng-change="addHighlightedRevision()" placeholder="12 character hg revision" ng-model="revisionToHighlight">
+        <div class="resetHighlightButton" ng-show="resetHighlightButton" ng-click="resetHighlight()">&#10006;</div>
```

Comment:
Likewise, you should be able to replace the resetHighlightButton variable with some kind of expression here. Don't know what it is off-hand but I'll leave it to you to figure out. :)